### PR TITLE
Meisam

### PR DESCRIPTION
**Pull request by Erwin6997**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,10 +12,97 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<!--navigation bar -->
```
Comment (ChrisOwen101) [inline index.html]: You can format the code in this file to make to easier to read by using the instructions here
https://stackoverflow.com/a/29973358

File: css/style.css [hunk 2]
```diff
@@ -1,19 +1,262 @@
...
+}
+
+.segments{
+    display: flex;
+    justify-content: space-evenly;
+    align-items: baseline;
+    text-align: center;
+}
+
+.nuzzle{
```
Comment (ChrisOwen101) [inline css/style.css]: All of these three classes have the same contents `width: 60%;`. It'd be better if you had one generic class

```
.segmentImage {
	width: 60%;
}
```
Then you can re-use the code which means you don't have to write so much!

File: index.html [hunk 3]
```diff
@@ -12,10 +12,97 @@
...
+
+ <!-- SEGMENTS -->
+<section class="seg">
+
+ <h2>
+    Everyone needs a little Karma.
+ </h2>
+ <div class="segments">
+    <div class="devices">
+        <img src="http://thinkful-ed.github.io/karma-clone/img/icon-devices.svg" alt="Devices" class="dev ">
```
Comment (ChrisOwen101) [inline index.html]: Rather than using a full URL for this image it's better to use a relative one 

```
../img/icon-devices.svg
````

**Comment by ChrisOwen101** [comment]
Great work!

File: css/style.css [hunk 4]
```diff
@@ -1,19 +1,262 @@
...
+	position: relative;
+}
+li {
+	list-style: none;
+}
+a {
+	color: Black;
+	text-decoration: none;
+}
+.container {
```
Comment (ChrisOwen101) [inline css/style.css]: Test

**Comment by Erwin6997** [comment]
thanks i fixed

**Pull request closed without merging**